baselines/dom.generated.d.ts

```diff
@@ -2404,89 +2404,89 @@ interface ANGLE_instanced_arrays {
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaRoleDescription) */
     ariaRoleDescription: string | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaRowCount) */
-    ariaRowCount: string | null;
+    ariaRowCount: `${bigint}` | "" | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaRowIndex) */
-    ariaRowIndex: string | null;
+    ariaRowIndex: `${bigint}` | "" | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaRowIndexText) */
     ariaRowIndexText: string | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaRowSpan) */
-    ariaRowSpan: string | null;
+    ariaRowSpan: `${bigint}` | "" | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaSelected) */
-    ariaSelected: string | null;
+    ariaSelected: ("true" | "false" | "undefined" | (string & {})) | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaSetSize) */
-    ariaSetSize: string | null;
+    ariaSetSize: `${bigint}` | "" | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaSort) */
-    ariaSort: string | null;
+    ariaSort: ("ascending" | "descending" | "none" | "other" | (string & {})) | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaValueMax) */
-    ariaValueMax: string | null;
+    ariaValueMax: `${number}` | "" | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaValueMin) */
-    ariaValueMin: string | null;
+    ariaValueMin: `${number}` | "" | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaValueNow) */
-    ariaValueNow: string | null;
+    ariaValueNow: `${number}` | "" | null;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/Element/ariaValueText) */
     ariaValueText: string | null;
-    role: string | null;
+    role: ("toolbar" | "tooltip" | "feed" | "math" | "presentation" | "none" | "note" | "application" | "article" | "cell" | "columnheader" | "definition" | "directory" | "document" | "figure" | "group" | "heading" | "img" | "list" | "listitem" | "meter" | "row" | "rowgroup" | "rowheader" | "separator" | "table" | "term" | "associationlist" | "associationlistitemkey" | "associationlistitemvalue" | "blockquote" | "caption" | "code" | "deletion" | "emphasis" | "insertion" | "paragraph" | "strong" | "subscript" | "superscript" | "time" | "scrollbar" | "searchbox" | "separator" | "slider" | "spinbutton" | "switch" | "tab" | "tabpanel" | "treeitem" | "combobox" | "menu" | "menubar" | "tablist" | "tree" | "treegrid" | "banner" | "complementary" | "contentinfo" | "form" | "main" | "navigation" | "region" | "search" | "alert" | "log" | "marquee" | "status" | "timer" | "alertdialog" | "dialog" | (string & {})) | null;
```

Comment:
Okay, this one is huge enough that I think exporting as an enum would be better.